index.html

```diff
@@ -420,6 +426,8 @@ <h3>Ecosystem Overview</h3>
           </figcaption>
         </figure>
 
+[PR NOTE. Figure 1 will need updating to replace Holder with Issuee/Holder]
+
```

Comment:
This note can be removed now, as Figure 1 has been updated.

Reply:
I'm a strong -1 to updating the diagram to introduce two terms for a single block in the diagram. This will confuse readers -- "Well, which one is it... 'holder' or 'issuee', or is it both?"

Note: This doesn't mean I'm averse to defining the term, but introducing it as a top-level term will harm more than it helps.